Packages/Core/Editor/Generator/Generator.cs

```diff
@@ -21,6 +22,191 @@ public static string[] GetTemplatePaths()
             return Directory.GetFiles(templateSearchPath, "UA_Template*.txt", SearchOption.AllDirectories);
         }
 
+        public static Dictionary<string, string> CreateTemplateVariablesMap(Type type, bool asPair, string withNamespace = default)
```

Comment:
Documentation would go a long way to explain the intent of the method.

Reply:
Samesies